book-my-show/backend/java/bms-monolith/src/main/java/org/lbcc/bms/bms_monolith/entity/SeatType.java

```diff
@@ -0,0 +1,25 @@
+package org.lbcc.bms.bms_monolith.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.lbcc.bms.bms_monolith.common.entity.BaseAuditingEntity;
+
+import java.math.BigDecimal;
+
+@Entity
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class SeatType extends BaseAuditingEntity {
+
```

Comment:
@SiZe(min = 4, max = 15, message = "Label should be between 4 and 15 characters")
@column(length = 15, nullable = false)
private String label;